src/main/java/flaxbeard/steamcraft/client/render/ItemSteamToolRenderer.java

```diff
@@ -63,15 +61,15 @@ public void renderItem(ItemRenderType type, ItemStack item, Object... data) {
         int use = 0;
         switch (this.type) {
```

Comment:
This switch statement seems unnecessary, you could just check the passed ItemStack for speed and use that.